analyticConfigurations/computed_metric-route53.json

```diff
@@ -0,0 +1,19 @@
+{
+  "analyticConfiguration": {
+    "metrics": [
+      {
+        "match": "aws.route53.healthcheckstatus.all",
+        "properties": {
+          "expression": "data.sum('aws.route53.healthcheckstatus.*') * 100 / data.count('aws.route53.healthcheckstatus.*')",
```

Comment:
Can you explain why this expression is dividing by different aggregations i.e. sum by a count?

Reply:
As the metric has only values 0 or 1, I use "sum" to calculate amount of healthy regions (amount of regions with value 1), and use "count" to calculate total amount of regions. So "sum" here represents count of "ones" :)
Please, see this link for the details: https://metricly.slack.com/archives/C1Y1R60MP/p1544634862142700
But it looks like CloudFront shows only global value for this metric, so there is currently only "us-east-1" which shows the value for all regions. I did some research in this thread: https://metricly.slack.com/archives/C1Y1R60MP/p1544719568168500?thread_ts=1544632419.136300&cid=C1Y1R60MP